src/Uno.UI/UI/Xaml/Internal/PointerCaptureTarget.cs

```diff
@@ -0,0 +1,60 @@
+﻿#nullable enable
+
+using System;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Uno.UI.Xaml.Core;
+
+internal class PointerCaptureTarget
+{
+	internal PointerCaptureTarget(UIElement element, PointerCaptureKind kind)
+	{
+		NativeCaptureElement = Element = element;
+		Kind = kind;
+	}
+
+	/// <summary>
+	/// The target element to which event args should be forwarded
+	/// </summary>
+	public UIElement Element { get; }
+
+	/// <summary>
+	/// The element to used for the native capture
+	/// </summary>
+	/// <remarks>
+	/// On WASM this might be different than the <see cref="Element"/>:
+	/// In case of implicit capture, the element used for the capture will prevent any pointer event on sub element
+	/// (sub element will actually get a pointer 'leave' on capture, and a 'enter' on capture release).
+	/// So instead of capturing using the actual element, we use the 'OriginalSource' of the 'relatedArgs',
+	/// so event will still be sent to sub elements and we will then filter them out if needed.
```

Comment:
```suggestion
	/// On WASM this might be different than the <see cref="Element"/>:
	/// In the case of implicit capture, the element used for the capture will prevent any pointer event on the sub-element
	/// (sub-element will actually get a pointer 'leave' on capture, and an 'enter' on capture release).
	/// So instead of capturing using the actual element, we use the 'OriginalSource' of the 'relatedArgs',
	/// so an event will still be sent to sub-elements and we will then filter them out if needed.
```